front/planning.php

```diff
@@ -128,8 +128,9 @@
             }
          }
 
-         if ($ismine || $canview) {
+         if (($ismine || $canview) && Session::startImpersonating($_GET["uID"], true)) {
```

Comment:
Maybe you can create a new method, like `Session::initForUser(int $user_id)` instead of using the impersonating feature which was not initially made for this kind of usages.